src/table/implementation.lisp

```diff
@@ -618,12 +625,23 @@
         (print-with-padding i j))
       (terpri output))))
 
+; Check if it is currently running in Jupyter.
+; Also see: https://github.com/yitzchak/common-lisp-jupyter
+(defun run-in-jupyter-p ()
+      (and (string= "JUPYTER" (package-name (symbol-package (type-of *standard-output*))))
+          (string= "IOPUB-STREAM" (symbol-name (type-of *standard-output*)))))
 
 (defmethod print-object ((object fundamental-table) stream)
   (if *print-pretty*
+    (if (run-in-jupyter-p)
+      ; Print 10 rows for jupyter output
+      (let ((string-stream (make-string-output-stream)))
```

Comment:
use with-output-to-string instead

Reply:
something like this:

```lisp
(with-output-to-string (string-stream)
    (show :html object :output string-stream :end 10)
    (funcall (find-symbol "HTML" :JUPYTER) (get-output-stream-string string-stream) :display t))
```

Reply:
just 
(funcall (find-symbol "HTML" :jupyter) (with-output-to-string (string-stream) ...) :display t)

also, this should be likely done even if *print-pretty* is nil, so maybe lift it up

Reply:
done, please review it again.

Reply:
Almost correct, but the (call-next-method) should be the default case in cond. Move it there, please.